app/views/users/newpost.html.erb

```diff
@@ -0,0 +1,7 @@
+Enter new user Information
+<hr>
+<%= form_with(model: [:users,@post ],url:"/users/#{@user_id}/newpost",format: :json) do |f| %>
+    Title: <%= f.text_field  :title %><br/>
+    Text: <%=f.text_field :text %><br/>
+    <button><% f.submit %>submit</button>
+<%end %>
```

Comment:
- [ ] I am not able to submit a new post to be able to check flash messages for `posts#create`, `comments#create`, and `likes#create` please address that